src/scripts/find-posts-by-hashtag.js

```diff
@@ -18,15 +13,17 @@ const findPostsByHashtag = async (hashtag) => {
 
     //read up to 3 pages of posts
     for (let i = 0; i < 3; i++) {
-      console.log(`Getting page #${i + 1}`);
-      const result = await profile.getUseCase('FindByHashtag').perform(
-        { hashtag, page },
-        {
-          provider,
-          parameters: {
-            accessToken,
-          },
-        }
+      console.error(`Getting page #${i + 1}`);
```

Comment:
Is this intentional that you use `console.error` to log this message?

Reply:
Yes, for capturing data from stdout, you don't want to get these diagnostic information into data. Additionally it's great for capturing just diagnostic data by doing `node script > /dev/null` – you still see messages from stderr (which is what `console.error` is for in Node)